composer.json

```
@@ -55,44 +48,106 @@
+            "BinaryConsumer\\": "example/binary/consumer/src",
+            "BinaryConsumer\\Tests\\": "example/binary/consumer/tests",
+            "BinaryProvider\\": "example/binary/provider/src",
+            "BinaryProvider\\Tests\\": "example/binary/provider/tests",
+            "MultipartConsumer\\": "example/multipart/consumer/src",
+            "MultipartConsumer\\Tests\\": "example/multipart/consumer/tests",
+            "MultipartProvider\\": "example/multipart/provider/src",
+            "MultipartProvider\\Tests\\": "example/multipart/provider/tests",
+            "XmlConsumer\\": "example/xml/consumer/src",
+            "XmlConsumer\\Tests\\": "example/xml/consumer/tests",
+            "XmlProvider\\": "example/xml/provider/src",
+            "XmlProvider\\Tests\\": "example/xml/provider/tests",
+            "MatchersConsumer\\": "example/matchers/consumer/src",
+            "MatchersConsumer\\Tests\\": "example/matchers/consumer/tests",
+            "MatchersProvider\\Tests\\": "example/matchers/provider/tests",
+            "GeneratorsConsumer\\": "example/generators/consumer/src",
+            "GeneratorsConsumer\\Tests\\": "example/generators/consumer/tests",
+            "GeneratorsProvider\\Tests\\": "example/generators/provider/tests",
+            "CsvConsumer\\": "example/csv/consumer/src",
+            "CsvConsumer\\Tests\\": "example/csv/consumer/tests",
+            "CsvProvider\\": "example/csv/provider/src",
+            "CsvProvider\\Tests\\": "example/csv/provider/tests",
+            "": [
+                "example/protobuf-sync-message/library/src",
+                "example/protobuf-async-message/library/src"
             ],
-            "MessageConsumer\\": [
-                "example/src/MessageConsumer",
-                "example/tests/MessageConsumer"
-            ],
-            "MessageProvider\\": [
-                "example/src/MessageProvider",
-                "example/tests/MessageProvider"
-            ]
+            "ProtobufSyncMessageConsumer\\": "example/protobuf-sync-message/consumer/src",
+            "ProtobufSyncMessageConsumer\\Tests\\": "example/protobuf-sync-message/consumer/tests",
+            "ProtobufSyncMessageProvider\\": "example/protobuf-sync-message/provider/src",
+            "ProtobufSyncMessageProvider\\Tests\\": "example/protobuf-sync-message/provider/tests",
+            "ProtobufAsyncMessageConsumer\\": "example/protobuf-async-message/consumer/src",
+            "ProtobufAsyncMessageConsumer\\Tests\\": "example/protobuf-async-message/consumer/tests",
+            "ProtobufAsyncMessageProvider\\Tests\\": "example/protobuf-async-message/provider/tests",
+            "StubServerConsumer\\": "example/stub-server/consumer/src",
+            "StubServerConsumer\\Tests\\": "example/stub-server/consumer/tests"
         }
     },
     "scripts": {
-        "start-provider": "php -S localhost:58000 -t example/src/Provider/public/",
```

TODO: check this is not referenced and is called out as deprecated. 

```suggestion
        "start-provider": "echo 'removed in 10.x'; exit 1",
```

---

Nothing references that I have seen, so maybe not needed. I Just appreciate QoL features like this.

---

Added again in https://github.com/pact-foundation/pact-php/pull/543